locator/topology.cc

```diff
@@ -167,15 +167,18 @@ void topology::sort_by_proximity(inet_address address, inet_address_vector_repli
 }
 
 int topology::compare_endpoints(const inet_address& address, const inet_address& a1, const inet_address& a2) const {
```

Comment:
I believe this particular comparer deserves its own personal unit test

Reply:
Maybe, though it's used on the fast path so any cluster test exercises it, in particular multi-dc ones.

Reply:
I think mistakes here won't show up in tests, as they end up sending requests to a node that is too far away rather than a node that doesn't hold data.

Reply:
I'm adding a unit test as we speak :)